### PR TITLE
(GH-40) added references

### LINKED ISSUE #40: Add references to CakeContrib.Guidelines and Cake.Addin.Analyzer
To further standardization between packages.

### PR DESCRIPTION
**Pull request by nils-a**
to Cake.Addin.Analyzer and CakeContrib.Guidelines
and the fixed the first two warnings:
Include PackageIcon and add TargetFramework net461

fixes #40

### CODE REVIEW

File: source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```
Comment (augustoproiete) [inline source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj:4]: LGTM in principle, but I wonder if adding `net461` would give bring any benefit on this particular addin given that it depends on NuGet.Protocol and friends, and they target `net472` and `netstandard2.0` only. Not against it though.

**Review by augustoproiete — APPROVED** [review]
LGTM. One minor comment on adding the `net461` target, but non-blocker.

File: source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj (hunk 1 above)
Comment (nils-a) [reply source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj:4]: "only" following https://cakebuild.net/docs/extending/addins/best-practices#net-target-version here 👼

**Comment by nils-a** [comment]
@augustoproiete I removed the double-targeting to `net461`

**Pull request merged**